tools/create-buildroot-image.sh

```diff
@@ -206,6 +208,26 @@ EOFEOF
 ;;
 esac
 
+# Adjust consts in buildroot source files.
+case "$TARGETARCH" in
+  arm64)
+    # 64 MB is too small for our large images.
+    sed -i 's/size = 64M/size = 256M/g' board/aarch64-efi/genimage-efi.cfg
+    # Also, use compressed images.
+    sed -i 's/Image/Image.gz/g' board/aarch64-efi/genimage-efi.cfg
+    # Overwrite grub.
+    cat >board/aarch64-efi/grub.cfg <<EOF
```

Comment:
The problem was that otherwise I'd have to copypaste more from the buildroot repo to this script and redo more of the work it already does.

I wrote a version of the script that does as you say - modifies nothing in the buildroot sources directly.
https://github.com/a-nogikh/syzkaller/commit/099a1278a364ae5003e57e947c5abe4013baff8c

Does it look better to you?

Reply:
That commit looks reasonable.
Are there any particular problems with it? I don't see lots of copypaste there.

Reply:
No particular problems. Applied the fix to this PR.